jans-cedarling/cedarling/src/jwt/mod.rs

```diff
@@ -56,6 +58,8 @@ pub enum JwtProcessingError {
     MissingClaimsInStrictMode(&'static str, &'static str),
     #[error("Failed to deserialize from Value to String: {0}")]
     StringDeserialization(#[from] serde_json::Error),
+    #[error("{0}")]
+    Unimplemented(String),
```

Comment:
looks like it is never used

Reply:
removed here [9bbf437](https://github.com/JanssenProject/jans/pull/10436/commits/9bbf437edb73a0ce4955603ffe3d191002ddeb0a)